### PR TITLE
Postman Tutorial update

### PR DESCRIPTION
**Pull request by bilo89za**
more updates ... hopefully we can merge this into master soon.

### CODE REVIEW

File: tapi/postman/README.md [hunk 1]
```diff
@@ -1,54 +1,37 @@
-![](logo.svg)
+![](./logo.png)
 
 # Postman Collection
 
 ## Overview
 
-This guide helps getting developers started with using the WhereIsMyTransport platform, using Postman. 
+This guide helps developers get started with the WhereIsMyTransport platform, using Postman.
```
Comment (ncthbrt) [inline tapi/postman/README.md]: Bit awkward phrasing

File: tapi/postman/README.md [hunk 2]
```diff
@@ -57,20 +40,20 @@ At the end of this guide you will have completed the following:
...
+1. Create a Client on the **WhereIsMyTransport** [Developer Portal](https://developer.whereismytransport.com/).
+    ![](devportal_client_create.png)
 
 2. Copy the *ClientId* and *ClientSecret* into the respective fields of the postman environment.
+    ![](devportal_client.png)
 
-![](devportal_client.png)
-
-3. From Postman, get a token, with the updated credentials, using the first API call in the collection, i.e. "**POST** Token". You should see a success response, along with a BearerToken response. This is automatically saved in the postman environment variables, and at this point you are authenticated to connect to the platform, until the token expires.
+3. From Postman, get a token, with the updated credentials, using the first API call in the collection, i.e. "**POST** token". You should see a success response, along with a BearerToken response. This is automatically saved in the postman environment variables, and at this point you are authenticated to connect to the platform, until the token expires.
```
Comment (ncthbrt) [inline tapi/postman/README.md]: Quite a long sentence. I tried to break it up a bit.

File: tapi/postman/README.md [hunk 3]
```diff
@@ -57,20 +40,20 @@ At the end of this guide you will have completed the following:
...
+    ![](postman_5_token.png)
 
-![](postman_5_token.png)
+4. To use the token for the other API calls, specify the token as a header with `Authorization: Bearer {token}`. For a complete set of the API's endpoints, please see the full [documentation](https://developer.whereismytransport.com/documentation). Below is an example of how to retrieve a list of agencies using the bearer token.
```
Comment (ncthbrt) [inline tapi/postman/README.md]: Would it be possible to make this maybe more explicit. I think people might get confused here

Comment (bilo89za) [inline tapi/postman/README.md]: I can add a screen shot here...

Comment (bilo89za) [inline tapi/postman/README.md]: wait, hang on... I already did add a screen shot, to make it clearer... can you suggest something...

**Pull request merged**